src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo__img {
+  display: flex;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
```

Comment:
The '.nav__list' class has a 'display: flex;' property which is good for horizontal alignment, but the vertical margins should be consistent. Consider adding a margin to the top or bottom for consistent spacing between elements.